BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb

```diff
@@ -829,11 +853,264 @@ $(function () {
       .toggle( $(this).prop('checked') );
   });
 
-  /* Clicking on a '-' button removes the row */
-  parameters.delegate('.tsk-prm-rm, .ui-icon-minus', 'click', function () {
-    $(this)
-      .closest('li')
-      .remove();
+  /* Preview Button */
+  $("#previewer").click(function () {
+    // Classes for the modal (popup)
+    var mdiv = 'preview-modal', mcont = 'preview-modal-content', mclose = 'preview-modal-close';
+    // Input parameters (fetch ids and data from cbcsvs and json descriptor)
+    var ids    = [<%= params.map    { |p| "\"#{p['id']}\""    }.join(", ") %>];
```

Comment:
Could any of these lists end up generating bad javascript? E.g. special characters in `p['name]'` or `c.name` ?